crates/ruff/src/rules/flake8_return/visitor.rs

```diff
@@ -32,14 +34,6 @@ impl<'a> ReturnVisitor<'a> {
                 }
                 return;
             }
-            ExprKind::Name { id, .. } => {
-                self.stack
-                    .assigns
-                    .entry(id)
-                    .or_insert_with(Vec::new)
-                    .push(expr.location);
-                return;
-            }
```

Comment:
Is this the same as the previous version? I know we have the logic in `// This is hacky ...`, but what if we visit a tuple here? Then we'd iterate over the names as part of _this_ method via the recursive call above. In the current version,  the `self.visit_assign_target` in the `Tuple` case above would have no effect, I think? Given `(a, b, c)`.

Reply:
I'm confused on what you mean here